MAD.Integration.TableauCRM/Startup.cs

```diff
@@ -55,18 +56,8 @@ public async Task PostConfigure(ConfigurationDbContext dbContext, IRecurringJobF
         {
             await dbContext.Database.MigrateAsync();
 
-            // Delete inactive jobs
-            foreach (var configuration in dbContext.Configuration.Where(y => y.IsActive == false))
```

Comment:
Restore this code otherwise jobs won't be created at start up.

Move the delete / register active jobs code into a service class so it can be reused here and in the JobManager job